test/tutorial/scripts/cli/put_file_cli.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-hca dss put-file --uuid 38f6c379-cb78-4a61-9310-f8cc0341c0eb --version 2019-07-30T164352.961501Z --creator-uid 0 --source-url s3://jeffwu-test/930a927d-0138-4a79-8c87-e45936fe4fc3/get_bundle.json
+hca dss put-file --uuid 38f6c379-cb78-4a61-9310-f8cc0341c0eb --version 2019-07-30T164352.961501Z --creator-uid 0 --source-url s3://dcp-cli-tutorials-put-file-cli/930a927d-0138-4a79-8c87-e45936fe4fc3/get_bundle.json
```

Comment:
is `dcp-cli-tutorials-put-file-cli` an actual bucket? we have a cli test bucket that can be used.

Reply:
Not an actual bucket, replaced this bucket with `org-humancellatlas-dss-cli-test` bucket